stboot.go

```diff
@@ -33,9 +33,10 @@ import (
 
 const (
 	hostCfgDefault = "/etc/host_configuration.json"
-	hostCfgHelp = `Location of Host Configuration file.
+	hostCfgHelp    = `Location of Host Configuration file.
 inside initramfs:       "/path/to/host_configuration.json"
 as efivar:              "efivar:YOURID-d736a263-c838-4702-9df4-50134ad8a636"
+as cdrom:               "cdrom:/path/to/host_configuration.json"
```

Comment:
Hi @bjoto, This patch looks great to me. Thanks for the linting fixes of my previous PR :sweat_smile:.
The only thing I'm not sure about is the prefix naming "cdrom". I understand that the Linux kernel uses the cdrom driver for ISOs and keeps this naming for legacy reason, but shouldn't we rather call it "iso" instead? I don't think there will ever be someone provisioning stboot with an actual CD-ROM. :smile:
What do you think about it ?

Reply:
Acutually, I'd prefer CD-ROM. It's more in line what it is. the /dev/sr0 is exactly a SCSI CDROM. Old, yes, but valid. :)

Reply:
IOW, I'd like to leave it as it is.

Reply:
sure, makes sense :grin: sound good to me